overlays/dev/csrf-secure-cookie-patch.yaml

```diff
@@ -1,14 +1,5 @@
 ---
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: tekton-dashboard
-  namespace: tekton-pipelines
-spec:
-  template:
-    spec:
-      containers:
-        - name: tekton-dashboard
-          env:
-            - name: CSRF_SECURE_COOKIE
-              value: "false"
+- op: add
+  path: /spec/template/spec/containers/0/args/-
```

Comment:
This is not targeting the correct container when oauth-proxy is added.

Reply:
@a-roberts should be fixed, can you retry please ?

Reply:
Thanks, confirmed that's good and the flag gets picked up perfectly now

Everything comes up and functions as I expect - testing read-only mode now